src/data-source.ts

```diff
@@ -0,0 +1,21 @@
+import "reflect-metadata";
+import { DataSource } from "typeorm";
+import config from "./config";
+
+export const AppDataSource = new DataSource({
+  type: "postgres",
+  host: config.DB_HOST,
+  port: 5432,
+  username: config.DB_USER,
+  password: config.DB_PASSWORD,
+  database: config.DB_NAME,
+  synchronize: true,
```

Comment:
This is wrong. Set `sychronize:false`. We only use `Migrations` for updating schema.

Reply:
updated @NdubuisiJr